frontend/app/components/api/api-work-packages/api-work-packages.service.ts

```diff
@@ -60,6 +63,24 @@ export class ApiWorkPackagesService {
   }
 
   /**
+   * Loads the work packages collection for the given work package IDs.
+   * Returns a WP Collection with schemas and results embedded.
+   *
+   * @param ids
+   * @return {WorkPackageCollectionResourceInterface}
+   */
+  public loadWorkPackagesCollectionFor(ids:string[]) {
+    return this.halRequest.get(
+      '/api/v3/work_packages',
+      {
+        filters: JSON.stringify([{ id: {operator: '=', values: ids }}])
+      },
```

Comment:
Not necessarily in this PR, but we should start using a service/helper for generating the filters.